File: tv/SampleTVApp/SampleTVApp/ViewController.m
//
//  ViewController.m
//  SampleTVApp
//
//  Created by Davis Koh on 9/18/15.
//  Copyright © 2015 com.DavisKoh. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    UIButton *button = [[UIButton alloc] initWithFrame:CGRectMake(500, 200, 500, 500)];
    [button setTitle:@"Hello Apple TV" forState:UIControlStateNormal];
    button.titleLabel.font = [button.titleLabel.font fontWithSize:50.0];
    
    [button addTarget:self
               action:@selector(buttonSelected:)
     forControlEvents:UIControlEventPrimaryActionTriggered];

    [self.view addSubview:button];
}

- (void)buttonSelected:(UIButton *)sender {
    NSLog(@"selected");
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: watch/SampleWatchApp/SampleWatchApp/ViewController.swift
//
//  ViewController.swift
//  SampleWatchApp
//
//  Created by Davis Koh on 5/17/16.
//  Copyright © 2016 com.DavisKoh. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()

    let label = UILabel(frame: CGRect(
      x: 50,
      y: 30,
      width: 220,
      height: 44))
    label.text = "meow"

    self.view.backgroundColor = UIColor.whiteColor()

    self.view.addSubview(label)
  }

  override func didReceiveMemoryWarning() {
    super.didReceiveMemoryWarning()
    // Dispose of any resources that can be recreated.
  }


}

